addons/website_slides/views/slide_channel_views.xml

```diff
@@ -8,7 +8,7 @@
             <field name="arch" type="xml">
                 <form string="Channels">
                     <header>
-                        <button name="action_channel_invite" string="Invite" type="object" class="oe_highlight"  attrs="{'invisible': [('enroll', '!=', 'invite')]}"/>
+                        <button name="action_channel_invite" string="Invite" type="object" class="oe_highlight" context="{'raise_on_not_allowed_to_add_members': True}" attrs="{'invisible': [('enroll', '!=', 'invite')]}"/>
```

Comment:
That's a long key ^^

Don't have any good proposals in mind, maybe "raise_if_no_members_access" ? @flch-odoo any suggestions?

Reply:
`slides_raise_if_not_allowed`?